sparkles/core.py

```diff
@@ -178,23 +198,25 @@ def run_aca_review(load_name=None, *, acas=None, make_html=True, outdir=None,
 
     # noinspection PyDictCreation
     if make_html:
+        from . import test
+        sparkles_version = test(get_version=True)
         context = {}
         context['load_name'] = load_name.upper()
         context['proseco_version'] = PROSECO_VERSION
-        context['aca_preview_version'] = ACA_PREVIEW_VERSION
-        context['acas'] = acas
-        context['summary_text'] = get_summary_text(acas)
+        context['sparkles_version'] = sparkles_version
+        context['acas'] = acars
```

Comment:
I don't know if this can be updated to use the key 'acars' due to the way the template is setup that this variable is being used for, but if it can handle an input of 'acars', then this should be updated, otherwise, a comment here noting the use in a template which treats these inputs as ACATables, would be good.